src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -836,6 +862,10 @@ $DOCKERFILE_SIGIL
 
     val tags = DefaultParsers.spaceDelimited("<tag>").parsed
 
+    if (!skipEcrLogin.value) {
```

Comment:
Because SBT. It would have to be a dynamic task, and at that point you can't pass an image name.

I could've created two tasks, one for the dependency image, and one for the main image tag - but that becomes far less readable and much more annoying if we need to add a new call.